indicators/outlook-hco41m.yml

```diff
@@ -0,0 +1,34 @@
+title: Outlook Phishing Kit hCO41m
+description: |
+    Detects a phishing kit pretending to be Outlook and attempting to capture the user's credentials.
+    Found as a result of this kit being deployed on Replit.
+
+
+references:
+    - https://urlscan.io/result/ef32cd01-2a2c-4513-bdc2-d44e7d3f870c/
+    - https://urlscan.io/result/686725c9-178d-494d-afbb-25900318cb70/
+
+detection:
+
+    title:
+      html|contains:
+        - <title>Document</title>
+
+    logo:
+      html|contains:
+        - <img src="logo.png" class="img-fluid">
+
+    privateComputerCheckmark:
+      html|contains:
+        - <img src="k.png" class="img-fluid">
+
+    harvester:
+      html|contains:
+        - https://esperanzahumura.org/d2/post.php
```

Comment:
The endpoint URL used as the credential harvesting server is trivial for this specific kit in my opinion, I think omitting it from the detection fields would improve the rule, you could possibly replace it with the service (`https://logo.clearbit.com`) the kit uses to dynamically put the targeted companies logo into the phishing page